core/src/main/java/com/amplifyframework/devmenu/DevMenuLogsFragment.java

```diff
@@ -39,10 +46,23 @@ public DevMenuLogsFragment() { }
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
-        View logsView = inflater.inflate(R.layout.dev_menu_fragment_logs, container, false);
+        logsView = inflater.inflate(R.layout.dev_menu_fragment_logs, container, false);
         // Display the logs (if any).
-        TextView logsText = logsView.findViewById(R.id.logs_text);
-        logsText.setText(DeveloperMenu.singletonInstance(getContext()).getLogs());
+        developerMenu = DeveloperMenu.singletonInstance(getContext());
+        logsText = logsView.findViewById(R.id.logs_text);
+        logsText.setText(developerMenu.getLogs());
+        // Search the logs when the search button is pressed.
+        logsView.findViewById(R.id.search_logs_button).setOnClickListener(view -> searchLogs());
         return logsView;
     }
+
+    /**
+     * Display the logs (if any) that contain the search query.
+     */
+    private void searchLogs() {
+        EditText searchText = logsView.findViewById(R.id.search_logs_field);
```

Comment:
Could you use a [SearchView](https://developer.android.com/reference/android/widget/SearchView) instead of an `EditText` here?  I believe it is basically just a wrapper around an `EditText`, but you get a lot of additional search functionality for free, like:
 - a search icon on the left side
 - a search button on the keyboard, which I believe is the more standard way for search to work, instead of having a button on the right side.
 - If your Android device has a hardware Search button, it'll actually work.

Reply:
I initially tried to use a SearchView instead of EditText, but the search icon that should be displayed to the left of the SearchView was not visible. The XML attribute used to set this icon is only available in API level 21+, yet Android still allocates space for the icon to the left of where the user would enter a query. Setting margin and padding to 0 does not remove the extra space for the icon. The result is shown in the picture below. I think having the extra space to the left of the SearchView looks strange and I could not find a way to either align the SearchView to the left or have an equal amount of space on either side of the SearchView.

<img width="414" alt="Screen Shot 2020-08-14 at 12 21 47 PM" src="https://user-images.githubusercontent.com/67125657/90286727-4b821e00-de2b-11ea-8213-26fea493d51a.png">

Reply:
Can you fix this by using `androidx.appcompat.widget.SearchView` instead of `android.widget.SearchView`?

Reply:
Yes, although it will require making Theme.AppCompat the parent style for the activity, which changes the styling of many components on the developer menu, but I should be able to fix the styling issues.